### PR TITLE
Fix executable issue: update api and add coveraga test

### PR DESCRIPTION
**Pull request by YingChen1996**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: src/promptflow/promptflow/_cli/_pf/_init_entry_generators.py [hunk 1]
```diff
@@ -232,11 +232,13 @@ def flow_inputs(self):
...
         )
-        return {flow_input: (value.default, value.type.value) for flow_input, value in executable.inputs.items()}
+        return {flow_input: (value.default, value.type.value) for flow_input, value in executable.inputs.items()
+                if flow_input != "chat_history"}
 
     @property
     def flow_inputs_params(self):
-        flow_inputs_params = ["=".join([flow_input, flow_input]) for flow_input, _ in self.flow_inputs.items()]
+        flow_inputs_params = ["=".join([flow_input, flow_input]) for flow_input, _ in self.flow_inputs.items()
+                              if flow_input != "chat_history"]
```
Comment (lalala123123) [inline src/promptflow/promptflow/_cli/_pf/_init_entry_generators.py:241]: The name of chat history may not be chat_history. Need change to
```
["=".join([flow_input, flow_input]) for flow_input, value in self.flow_inputs.items() if not value.is_chat_history]
```

Comment (YingChen1996) [reply src/promptflow/promptflow/_cli/_pf/_init_entry_generators.py:241]: Thanks, @brynn-code, according to the discussion yesterday, may I ask we ignore `chat_history` input or all flow inputs with `is_chat_history` is True?

Comment (brynn-code) [reply src/promptflow/promptflow/_cli/_pf/_init_entry_generators.py:241]: Yeah we may support chat history in the next pr, no rush to this one, as our test page (the html one) has a hard code key for now. I remember we have a task, we may use that to track this.

Comment (YingChen1996) [reply src/promptflow/promptflow/_cli/_pf/_init_entry_generators.py:241]: keep `flow_input != "chat_history"` behavior for now

**Review by lalala123123 — DISMISSED** [review]

File: src/promptflow/promptflow/_sdk/data/executable/main.py.jinja2 [hunk 2]
```diff
@@ -9,7 +9,7 @@ from bs4 import BeautifulSoup, NavigableString, Tag
 
 from promptflow._sdk._utils import print_yellow_warning
 from promptflow._sdk._serving.flow_invoker import FlowInvoker
-from promptflow._utils.multimedia_utils import is_multimedia_dict, MIME_PATTERN
```
Comment (lumoslnt) [inline src/promptflow/promptflow/_sdk/data/executable/main.py.jinja2:12]: Please do not change the function name here, I have changed the function name in multimedia_utils.

File: src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py [hunk 3]
```diff
@@ -1119,6 +1120,42 @@ def get_node_settings(_flow_dag_path: Path):
...
+                process = subprocess.Popen(['python', app_file])
+
+                try:
+                    # Wait for a specified time (in seconds)
+                    wait_time = 5
+                    process.wait(timeout=wait_time)
+                    if process.returncode == 0:
+                        pass
+                    else:
+                        raise Exception(f"Process terminated with exit code {process.returncode}")
```
Comment (wangchao1230) [inline src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py:1151]: will the ci log contain the error message in case of fail?

**Review by wangchao1230 — APPROVED** [review]

File: src/promptflow/promptflow/_sdk/data/executable/main.py.jinja2 (hunk 2 above)
Comment (YingChen1996) [reply src/promptflow/promptflow/_sdk/data/executable/main.py.jinja2]: revert

File: src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py (hunk 3 above)
Comment (YingChen1996) [reply src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py:1151]: not for now ,will only show

Comment (YingChen1996) [reply src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py:1151]: will refine in the next pr

Comment (YingChen1996) [reply src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py:1151]: update： 
image

**Review by wangchao1230 — APPROVED** [review]

**Review by lalala123123 — APPROVED** [review]

**Pull request merged**